### PR TITLE
addAppliedDiscounts and removeAppliedDiscounts

### PR DESCRIPTION
**Pull request by ellismarte**

### CODE REVIEW

**Comment by ellismarte** [comment]
I think this is failing because it depends on the PR I just made for bradibble.js

File: src/actions/user/loyalties.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint no-shadow:1 */
```
Comment (sepowitz) [inline src/actions/user/loyalties.js]: Why aren't we doing this in ```actions/session/user```?

File: src/reducers/error.js [hunk 2]
```diff
@@ -88,6 +88,11 @@ import {
   REMOVE_ALLERGENS,
 } from '../actions/session/user';
 
+// loyalties
```
Comment (sepowitz) [inline src/reducers/error.js]: We should add the ```add``` and ```remove``` discount actions to the errors reducer

File: src/reducers/user/loyalties.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { FETCH_CUSTOMER_LOYALTIES } from '../../actions/user/loyalties';
+
+export const initialState = {
+  loyalties: [],
+};
+
+export default (state = initialState, action) => {
+  const { payload, type } = action;
+  let newState;
```
Comment (sepowitz) [inline src/reducers/user/loyalties.js]: no need for this

File: src/actions/user/loyalties.js (hunk 1 above)
Comment (ellismarte) [reply src/actions/user/loyalties.js]: mmm it looks like we are, I'm guessing this is redundant? Removing now.

File: src/reducers/user/loyalties.js (hunk 3 above)
Comment (ellismarte) [reply src/reducers/user/loyalties.js]: no need for newState? removing now.

**Pull request merged**